layers/core_checks/cc_render_pass.cpp

```diff
@@ -882,7 +871,23 @@ bool CoreChecks::PreCallValidateCmdEndRenderPass2KHR(VkCommandBuffer commandBuff
 
 bool CoreChecks::PreCallValidateCmdEndRenderPass2(VkCommandBuffer commandBuffer, const VkSubpassEndInfo *pSubpassEndInfo,
                                                   const ErrorObject &error_obj) const {
-    return ValidateCmdEndRenderPass(commandBuffer, pSubpassEndInfo, error_obj);
+    bool skip = false;
+    skip |= ValidateCmdEndRenderPass(commandBuffer, error_obj);
+
+    const auto &cb_state = *GetRead<vvl::CommandBuffer>(commandBuffer);
+    const auto *rp_state_ptr = cb_state.active_render_pass.get();
+
+    if (pSubpassEndInfo && pSubpassEndInfo->pNext) {
```

Comment:
```suggestion
    if (pSubpassEndInfo) {
```

because `FindStructInPNextChain` checks first for null anyway